members/CusB94a1a75c1534267461.yaml

```diff
@@ -0,0 +1,131 @@
+bioguide: CusB94a1a75c1534267461
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://www.regulations.gov/comment?D=CEQ-2018-0001-0001"
+    - wait:
+      - value: 6
+    - find:
+      - selector: input#gwt-uid-164
+    - click_on:
+        - selector: input#gwt-uid-164
+    - wait:
+      - value: 1
+    - find:
+      - selector: textarea#x-auto-0-input
+    - fill_in:
+        - name: message
+          selector: textarea#x-auto-0-input
+          value: $MESSAGE
+          required: true
+        - name: First Name
+          selector: input#x-auto-1-input
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: input#x-auto-2-input
+          value: $NAME_LAST
+          required: true
+    - click_on:
+        - selector: input#gwt-uid-85
```

Comment:
to simplify this yaml a bit, could you put this click_on alongside the other one at the top?